routing/loaded_path_segment.hpp

```diff
@@ -38,21 +36,14 @@ struct LoadedPathSegment
   TNodeId m_nodeId;
   vector<SingleLaneInfo> m_lanes;
 
-  // General constructor.
-  LoadedPathSegment(RoutingMapping & mapping, Index const & index,
-                    RawPathData const & osrmPathSegment);
-  // Special constructor for side nodes. Splits OSRM node by information from the FeatureGraphNode.
-  LoadedPathSegment(RoutingMapping & mapping, Index const & index,
-                    RawPathData const & osrmPathSegment, FeatureGraphNode const & startGraphNode,
-                    FeatureGraphNode const & endGraphNode, bool isStartNode, bool isEndNode);
-
-private:
-  // Load information about road, that described as the sequence of FtSegs and start/end indexes in
-  // in it. For the side case, it has information about start/end graph nodes.
-  void LoadPathGeometry(buffer_vector<TSeg, 8> const & buffer, size_t startIndex,
-                        size_t endIndex, Index const & index, RoutingMapping & mapping,
-                        FeatureGraphNode const & startGraphNode,
-                        FeatureGraphNode const & endGraphNode, bool isStartNode, bool isEndNode);
+  LoadedPathSegment(TEdgeWeight weight, TNodeId nodeId)
```

Comment:
Please, re-order fields in this class from the most heavy (vectors and strings) to the most lightweight (bools).